unasync/__init__.py

```diff
@@ -64,21 +64,21 @@ def untokenize(tokens):
     return ''.join(space + tokval for space, tokval in tokens)
 
 
-def bleach(filepath, fromdir, todir):
+def unasync(filepath, fromdir, todir):
     with open(filepath, 'rb') as f:
         encoding, _ = std_tokenize.detect_encoding(f.readline)
         f.seek(0)
         tokens = tokenize(f)
-        tokens = bleach_tokens(tokens)
+        tokens = unasync_tokens(tokens)
         result = untokenize(tokens)
         outfilepath = filepath.replace(fromdir, todir)
         os.makedirs(os.path.dirname(outfilepath), exist_ok=True)
         with open(outfilepath, 'w', encoding=encoding) as f:
             print(result, file=f, end='')
 
 
-class bleach_build_py(build_py):
-    """Monkeypatches build_py to add bleaching from _async to _sync"""
+class build_py(build_py):
+    """Monkeypatches build_py to add unasyncing from _async to _sync"""
```

Comment:
"Copies files from _async to _sync and removes async/await tokens" is maybe clearer? Not sure.

Reply:
How about convert files in _async dir from being asynchronous to synchronous and saves them in _sync dir.